src/components/Layout/RightSidebar.tsx

```diff
@@ -321,6 +321,34 @@ const RightSidebar = () => {
                 </Tooltip>
               </a>
             ))}
+            <a
+              href={`${location.pathname.replace(/\/$/, '')}/index.md`}
+              className="flex h-5 ui-theme-dark group/markdown-link cursor-pointer"
+              onClick={(e) => {
+                // In development mode, markdown files aren't generated, so show alert immediately
+                if (process.env.NODE_ENV === 'development') {
+                  e.preventDefault();
+                  alert('Markdown files are only available in production builds. Run "yarn build" to generate them.');
```

Comment:
Alerts are pretty poor UX, but given this is just for development, it's not a view I'll push strongly here. I would clarify that `yarn serve` is needed in addition to `yarn build` to run a local "production" version